Makefile

```diff
@@ -50,12 +50,25 @@ operator-local-gcp:
 	@$(MAKE) operator-stop-gcp || true
 	@./dev/operator_local.sh --operator-only --gcp || true
 
+# start local operator and attach the delve debugger to it (in server mode)
+operator-local-aws-dbg:
```

Comment:
Convert `operator-local-aws-dbg` -> `operator-local-dbg-aws`. Likewise for the GCP one.